src/helpers/reader.js

```diff
@@ -59,7 +59,7 @@ function readFile(file, next) {
   fs.readFile(file, 'utf8', (err, data) => {
 
     if (err) {
-      log.error(err);
+
```

Comment:
I don't really care how it works in other parts of the code. I care about this particular use case and in this use case it is not needed to log, it may only confuse the user. In Node you should propagate error and not log it everywhere. Btw "in most" means basically that we are inconsistent already :)